internal/convert/pod.go

```diff
@@ -104,11 +108,19 @@ func toPodTemplate(serviceConfig latest.ServiceConfig, labels map[string]string,
 	return tpl, nil
 }
 
-func toImagePullPolicy(image string) apiv1.PullPolicy {
-	if strings.HasSuffix(image, ":latest") {
-		return apiv1.PullAlways
+func toImagePullPolicy(image string, specifiedPolicy string) (apiv1.PullPolicy, error) {
```

Comment:
Do we want to introduce a cross-package dependency for this ? (5 lines of code)
I certainly don't want the installer to depend on the `convert` package (even if it actually does for the ugly CRD to API server migration related stuff), and it would feel strange to have the `convert` package depends on the installer. Should we introduce a package for that ?

Reply:
Didn't realise it would add that as a dependency. It's simple enough that we can just leave this duplication. It's unlikely that the pull policies will change.